src/katgpucbf/fgpu/accum.py

```diff
@@ -17,39 +17,34 @@
 """Compute statistics by bucketing into windows."""
 
 from dataclasses import dataclass
-from typing import Final, Generic, Protocol, Self, TypeVar
-
-_T_co = TypeVar("_T_co", covariant=True)
```

Comment:
I don't pretend to understand covariance and contravariance, but is this "distinction" (?) not required anymore?

Reply:
One of the features of PEP 695 is that it automatically infers whether a generic variable is covariant and/or contravariant based on how it is used.